guidelines/index.html

```diff
@@ -1033,21 +1031,21 @@ <h4 id="x0-5-1-new-features-in-wcag-2-1"><span class="secno">0.5.1 </span>WCAG 2
 						<li>4.1.3 <a href="#status-messages">ステータスメッセージ</a> (AA)</li>
 					</ul>
 
-					<p>これらの達成基準の多くは、新しい用語を参照している。その新しい用語はまた、用語集に追加されており、かつ達成基準の規定要件の一部を構成している。</p>
+					<p>新しい達成基準は、新しい用語を参照していることがある。その新しい用語はまた、用語集に追加されており、かつ達成基準の規定要件の一部を構成することがある。</p>
```

Comment:
これでも良いとは思うのですが、原文を見ると1文になっていて、無理に分割しなくても成立するように思いました。

>The new success criteria may reference new terms that have also been added to the glossary and form part of the normative requirements of the success criteria.

たとえば以下のようにあっさりやってしまっても良いのではないかと思いますが、どうでしょうか。alsoの位置が微妙なのですが、意味としてはこういうことだと思います。

```suggestion
					<p>新しい達成基準は、用語集に追加された新しい用語を参照していることがあり、それらもまた達成基準の規定要件の一部を構成することがある。</p>
```